extension/test/unit/checkout/finishCheckout-test.js

```diff
@@ -0,0 +1,50 @@
+const assert = require('assert')
+const executeStep = require('../../../checkout/finishCheckout')
+
+describe('backupCheckout', () => {
+  const checkout = {
+    user: {
+      id: 'sdsd-ddf-dff',
+      mail: 'mail@mail.com'
+    },
+    items: [{
+      id: 'qwerty123',
+      name: 'QWERTY123',
+      unitPrice: 100,
+      quantity: 1
+    }]
+  }
+  const context = {
+    log: {
+      error: () => {}
+    },
+    storage: {
+      user: {}
+    }
+  }
+
+  it('Should backup checkout and remove original checkout', async () => {
+    const orderId = 'ererer-rtrtrt-tytyty'
+
+    context.storage.user.get = async (key) => {
+      assert.equal(key, 'checkout')
```

Comment:
This is too much going on for a single test. Every assert could be one single test.